pkg/logs/tailers/socket/tailer.go

```diff
@@ -67,7 +68,16 @@ func (t *Tailer) forwardMessages() {
 	}()
 	for output := range t.decoder.OutputChan {
 		if len(output.GetContent()) > 0 {
-			t.outputChan <- message.NewMessageWithSource(output.GetContent(), message.StatusInfo, t.source, output.IngestionTimestamp)
+			origin := message.NewOrigin(t.source)
+			remote_address := t.Conn.RemoteAddr()
```

Comment:
style nit: can we change the variable names to use [mixed caps](https://go.dev/doc/effective_go#mixed-caps) ?